### PR TITLE
Logo updates

### PR DESCRIPTION
**Pull request by ajanes3**
Updated WJAR logo
Updated KDKA logo
Updated WFSB logo
Updated WTIC logo
Updated WVIT logo
Updated WGGB logo
Updated WJZ logo
Updated WMAR logo
Updated WMYT logo
Updated WNBC logo
Updated WABC logo
Updated WCBS logo
Updated WBTS logo
Updated WVUE logo
Updated WTMJDT2 logo

### CODE REVIEW

**Review by LaneSh4d0w — DISMISSED** [review]

File: data/channels.csv [hunk 1]
```diff
@@ -25690,7 +25690,7 @@ WBSTV.kr,WBS TV,원음방송,,,KR,KR-11,,c/KR;s/KR-11,kor,religious,FALSE,,,,htt
 WBSTV.ug,WBS TV,,,,UG,,,c/UG,eng,,FALSE,,,,http://wbs.ug/,https://i.imgur.com/ynKBHPh.png
 WBTDLD2.us,WBTD-LD2,,Bounce,,US,US-VA,Suffolk,s/US-VA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/e/e0/Bounce_TV_logo.svg/512px-Bounce_TV_logo.svg.png
 WBTDLD3.us,WBTD-LD3,,H&I,,US,US-VA,Suffolk,s/US-VA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/9/9e/Heroes_%26_Icons_logo.svg/512px-Heroes_%26_Icons_logo.svg.png
-WBTSCD1.us,WBTS-CD1,,NBC,,US,US-NH,Nashua,s/US-NH,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/NBC_Peacock_1986.svg/512px-NBC_Peacock_1986.svg.png
+WBTSCD1.us,WBTS-CD1,,NBC,,US,US-NH,Nashua,s/US-NH,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/c/c6/WBTS-LD_NBC_10_Boston_logo.png
```
Comment (freearhey) [inline data/channels.csv:25693]: Image size should not be more than 512x512 px.

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels

File: data/channels.csv [hunk 2]
```diff
@@ -27196,7 +27196,7 @@ WJACDT4.us,WJAC-DT4,,CW+,,US,US-PA,Johnstown,s/US-PA,eng,,FALSE,,,,,https://uplo
 WJALDT1.us,WJAL-DT1,,SonLife,,US,US-PA,Chambersburg,s/US-PA,eng,,FALSE,,,,http://www.wjal.com/,https://i.imgur.com/JfRqVn6.png
 WJANCD1.us,WJAN-CD1,,América TeVé,,US,US-FL,Miami,s/US-FL,eng,,FALSE,,,,,https://i.imgur.com/zfurFwW.png
 WJANLD1.us,WJAN-LD1,,América TeVé,,US,US-FL,Miami,s/US-FL,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/a/af/Am%C3%A9rica_Tev%C3%A9_Logo.png
-WJARDT1.us,WJAR-DT1,,NBC,,US,US-RI,Cranston,s/US-RI,eng,,FALSE,,,,https://turnto10.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/7a/NBC_logo_2022_%28vertical%29.svg/512px-NBC_logo_2022_%28vertical%29.svg.png
+WJARDT1.us,WJAR-DT1,,NBC,,US,US-RI,Cranston,s/US-RI,eng,,FALSE,,,,https://turnto10.com/,https://upload.wikimedia.org/wikipedia/commons/8/8f/WJAR_LOGO_NBC10_Solid_Calls_vert_blu.png
```
Comment (freearhey) [inline data/channels.csv:27199]: Logo is too big.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked logos again.

File: data/channels.csv (hunk 1 above)
Comment (ajanes3) [reply data/channels.csv:25693]: fixed in latest update

File: data/channels.csv (hunk 2 above)
Comment (ajanes3) [reply data/channels.csv:27199]: fixed in latest update

**Review by freearhey — APPROVED** [review]

**Pull request merged**